Mail/Views/Alerts/CreateFolderView.swift

```diff
@@ -37,10 +37,13 @@ struct CreateFolderView: View {
 
     @State private var folderName = ""
     @State private var error: FolderError?
+    @State private var isModifyView = false
+    @State private var isInitialName = false
 
     @FocusState private var isFocused
 
     let mode: Mode
+    let folder: Folder?
```

Comment:
In an other comment I suggested adding an init. In the init you can use a default arg